perfkitbenchmarker/configs/benchmark_config_spec.py

```diff
@@ -341,7 +433,8 @@ def _GetOptionDecoderConstructions(cls):
     result.update({
         'description': (option_decoders.StringDecoder, {'default': None}),
         'flags': (_FlagsDecoder, {}),
-        'vm_groups': (_VmGroupsDecoder, {})})
+        'vm_groups': (_VmGroupsDecoder, {'default': None}),
```

Comment:
Making the default an empty dictionary would save having to check self.vm_groups in a few places (line 406 above and line 205 in benchmark_spec.py)